appengine/building-an-app/update/server.js

```diff
@@ -14,42 +14,42 @@
 
```

Comment:
Update copyright year to this year ☝🏼 too  https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/3617/checks?check_run_id=21334109681

Reply:
In general, we shouldn't be updating the copyright for files is my understanding.

Reply:
Required of `header-check` bot if a file is changed to also update the year.

Reply:
sorry for the long delay in responding, just checked the configuration and it seems like license year is ignored in this repo. go/copyright for policy, but in general we shouldn't be updating copyright year on updating a file (including if we move the file)

Reply:
The problem was with the Google Inc versus Google LLC